### PR TITLE
Consoliate bibliography files and create initial publications page

### LINKED ISSUE #127: Bibliographic information in TMAP8 should be consolidated and a publications page created
## Reason
Bibliographic information in TMAP8 documentation is scattered across multiple `.bib` files. This should not be encouraged, in my opinion, as it leads to opportunities for multiple bibliographic entries and general confusion about what is already available to cite.

Also, with the publishing of https://doi.org/10.1016/j.fusengdes.2024.114438, bibliographic information related to publications using TMAP8 should be tracked in a publications page.

## Design
Combine all bib files into the already created `doc/content/bib/tmap8.bib` file. Create a publications markdown page with its own bibliography file.

## Impact
Easier management of bibliographic entries in TMAP8 documentation, better tracking for our publications and impact.

### PR DESCRIPTION
**Pull request by cticenhour**
Closes #127

Requesting review from @simopier

### CODE REVIEW

File: doc/content/publications.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+!bibtex list bib_files=articles_using_tmap8.bib
+
+!! The below commented out sections should be added in as publications are created and available
+
+!! ## Technical Reports
+
+!! !bibtex list bib_files=tmap8_technical_reports.bib
+
+!! ## Theses and Dissertations)
```
Comment (cticenhour) [inline doc/content/publications.md:17]:
```suggestion
!! ## Theses and Dissertations
```

Need to remove this after review.

File: doc/content/verification_and_validation/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ For [software quality assurance](sqa/index.md) purposes, TMAP8 undergoes verific
 
 Note that in addition to monitoring TMAP8 performance and reproducibility in verification and validation cases, the effects of changes made to TMAP8 are tracked. A series of automated tests are performed via continuous integration using [CIVET](https://civet.inl.gov/repo/530) to help identify any changes in TMAP8's predictions, therefore ensuring stability and robustness.
 
-TMAP8 also contains [example cases](examples/index.md), which showcase how TMAP8 can be used.
+TMAP8 also contains [example cases](examples/index.md), which showcase how TMAP8 can be used. Finally, for more references of TMAP8 usage, a list of publications that use TMAP8 can be found [here](publications.md).
```
Comment (simopier) [inline doc/content/verification_and_validation/index.md:5]:
```suggestion
TMAP8 also contains [example cases](examples/index.md), which showcase how TMAP8 can be used. Finally, for more references of TMAP8 usage, a list of publications supporting TMAP8 development can be found [here](publications.md).
```

File: doc/content/publications.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# List of Publications using TMAP8
+
+Below is a known list of publications using TMAP8, and might be incomplete. If
```
Comment (simopier) [inline doc/content/publications.md:3]:
```suggestion
Below is a known list of publications supporting TMAP8 development, and might be incomplete. If
```

**Review by simopier — CHANGES_REQUESTED** [review]
Thank you @cticenhour, I just have a small comment.

**Comment by cticenhour** [comment]
@simopier I accept your suggestions - ready for another look!

**Review by simopier — APPROVED** [review]
Thank you, @cticenhour!!

**Pull request merged**